schemas/webpackOptionsSchema.json

```diff
@@ -76,7 +76,14 @@
         },
         {
           "additionalProperties": {
-            "type": "string"
+            "anyOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "object"
+              }
+            ]
           },
```

Comment:
Interesting, we don't use that anywhere at the moment in the schema. Will try out if it works.